src/utils/constants.ts

```diff
@@ -87,7 +87,7 @@ export const URL_REGEX = new RegExp(
   'i',
 );
 export const IO_PROCESS_ID =
-  process.env.VITE_IO_PROCESS_ID || ioDevnetProcessId;
+  process.env.VITE_IO_PROCESS_ID || arioDevnetProcessId;
```

Comment:
Suppose should change this to VITE_ARIO_PROCESS_ID and update github actions and env vars

Reply:
As well as change the constant to ARIO_PROCESS_ID